packages/compass-schema/src/components/coordinates-minichart/utils.js

```diff
@@ -1,5 +1,4 @@
 import L from 'leaflet';
-import fetch from 'node-fetch';
```

Comment:
`node-fetch` was not working because it depends on outdated version of whatwg-url that uses sharedarraybuffer that doesn't exist in browser environment anymore. As this code is running in the browser, there is no need to use `node-fetch` anyway